spec/helpers/users_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
- [ ] Considering you were advised not to use the generate command, so that you can get a hands-on approach to how routing and controllers work. But you used them anyway. Kindly remove this generated file.